tests/ValidatorTest.php

```diff
@@ -245,4 +245,13 @@ public function test_iban_validation_results_in_InvalidChecksumException(): void
 
         $this->validator->validate(new Iban('DE90 3704 0044 0532 0130 00'), throw: true);
     }
+    
+    public function test_validations_are_cleared_before_validation(): void
+    {
+        $this->validator->validate(new Iban('DE89 3704 0044 053A 013B 00'));
+
```

Comment:
Let's add one more assertion:

```suggestion
$this->assertFalse($isValid);
```

Reply:
done